### ISSUE TITLE
heketi server doesn't log errors on volume create failures

### ISSUE THREAD

**Issue by MohamedAshiqrh** [heketi/heketi#609]
heketi-cli seems to correctly print the proper error message to stdout now. But the same should be logged in the server logs as well.

```
{"log":"[negroni] Started POST /volumes\n","stream":"stdout","time":"2016-11-24T11:04:11.642199782Z"}
{"log":"[negroni] Completed 400 Bad Request in 215.879µs\n","stream":"stdout","time":"2016-11-24T11:04:11.642272341Z"}
```

**Comment by MohamedAshiqrh** [comment]
Heketi logs
```
Heketi v3.0.0-66-gf1e76e8-log2
[heketi] INFO 2016/12/06 13:16:12 Loaded mock executor
[heketi] INFO 2016/12/06 13:16:12 Loaded simple allocator
[heketi] INFO 2016/12/06 13:16:12 GlusterFS Application Loaded
Listening on port 3333
[negroni] Started POST /volumes
[heketi] DEBUG 2016/12/06 13:17:22 /src/github.com/heketi/heketi/apps/glusterfs/volume_entry.go:84: [943c4d34bf9530ab3b43ce4081c533d0] Replica 3
[heketi] ERROR 2016/12/06 13:17:22 /src/github.com/heketi/heketi/apps/glusterfs/app_volume.go:145: Requested volume size (2 GB) is smaller than the minimum supported volume size (4194304)
[negroni] Completed 400 Bad Request in 201.944µs

```
Heketi-cli
```

heketi-cli -s http://localhost:3333 volume create --size=2 
Error: Requested volume size (2 GB) is smaller than the minimum supported volume size (4194304)
```

**Comment by obnoxxx** [comment]
closed by merging #610
